Jenkinsfile

```diff
@@ -51,15 +48,14 @@ pipeline {
             }
         }
 
-        stage('Run install') {
-            steps {
-                sh './mvnw org.jacoco:jacoco-maven-plugin:0.8.3:prepare-agent clean install org.jacoco:jacoco-maven-plugin:0.8.3:report coveralls:report --quiet --batch-mode -nsu'
+        stage('Install & Test on JDK 1.8') {
+            tools {
+                jdk 'JDK 1.8 (latest)'
             }
-        }
 
-        stage('Run install') {
             steps {
-                sh './mvnw javadoc:javadoc -Dmaven.test.skip=true --quiet'
+                sh './mvnw org.jacoco:jacoco-maven-plugin:0.8.3:prepare-agent clean install org.jacoco:jacoco-maven-plugin:0.8.3:report coveralls:report'
```

Comment:
Let's make it run for a while and consider adding `--quiet` to depress some verbose logs

Reply:
In jenkins, I think we could keep this.